Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/AgendaDayFragment.java

```diff
@@ -80,14 +98,40 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         ButterKnife.bind(this, view);
 
         recycler.setLayoutManager(new LinearLayoutManager(getActivity().getApplicationContext()));
-        setupHeaderAdapter();
+        fetchScheduleData(dayFilter);
 
         return view;
     }
 
-    private void setupHeaderAdapter() {
-        List<ScheduleDatabase.ScheduleRow> rows = ScheduleDatabase
-                .fetchScheduleListByDay(getActivity().getApplicationContext(), dayFilter);
+    private void fetchScheduleData(String dayFilter) {
+        firebaseHelper.getMainDatabase().child("conferenceData").child("events")
```

Comment:
might be worth putting this into firebase helper?
  firebaseHelper.getMainDatabase().child("conferenceData").child("events")

Reply:
done